DungeonMaster/Assets/Scripts/NetworkingCalls.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+
+public class NetworkingCalls : MonoBehaviour
+{
+    IEnumerator GET(string url, string requestFunction, Type type) {
+        UnityWebRequest www = UnityWebRequest.Get(url);
+        www.downloadHandler = (DownloadHandler)new DownloadHandlerBuffer();
+        
+        GlobalDebug.LogMessage("function: " + requestFunction + " waiting for request: " + www.url);
+        yield return www.SendWebRequest();
+        if (www.isNetworkError || www.isHttpError) {
+            GlobalDebug.LogMessage("request failed, url: " + www.url + " error: " + www.error + " body: " + System.Text.Encoding.UTF8.GetString(www.uploadHandler.data));
+        } else {
+            GlobalDebug.LogMessage("request succeeded, url: " + www.url + " responseCode: " + www.responseCode + " body: " + www.downloadHandler.text);
+        }
+        JsonUtility.FromJson<Type>(www.downloadHandler.text);
+    }
+    
+    string GetJSONWithoutID(object o) {
+        string json = JsonUtility.ToJson(o);
+        return json.Replace("\"\"", "null");
```

Comment:
Does this technically still pass a string with the value `null` in the field your replacing? Was that the intent?

Reply:
https://github.com/oneleif/DungeonMaster-Client/issues/16